capnpc-go/templates/interfaceClient

```diff
@@ -21,14 +21,36 @@ func (c {{$.Node.Name}}) {{.Name|title}}(ctx {{$.G.Imports.Context}}.Context, pa
 }
 {{end}}
 
+// String returns a string that identifies this capability for debugging
+// purposes.  Its format should not be depended on: in particular, it
+// should not be used to compare clients.  Use IsSame to compare clients
+// for equality.
+func (c {{$.Node.Name}}) String() string {
+	return capnp.Client(c).String()
```

Comment:
I think it would be a bit better to return something that reflects the type of the capability, rather than just shelling out to untyped Client. Maybe`fmt.Sprintf("%T(%v)", c, capnp.Client(c))`